backend/dataall/core/permissions/db/tenant/tenant_repositories.py

```diff
@@ -1,6 +1,6 @@
 import logging
 
-from dataall.core.permissions.db import permission_models as models
+from dataall.core.permissions.db.tenant import tenant_models as models
 
```

Comment:
In all the functions of `Tenant` we should not be validating the input (`if note tenant_name....`). That should happen in the `api/resolver`

Reply:
Yes, It's rudimental check. Everything is already checked in services.
As for "save_tenant" it's only used with 'dataall' constant;